yt_dlp/extractor/_extractors.py

```diff
@@ -491,7 +491,6 @@
     FancodeVodIE,
     FancodeLiveIE
 )
-
```

Comment:
revert. This will cause unnecessary conflicts

Reply:
Fixed